wpilib/tests/test_robotdrive.py

```diff
@@ -2,6 +2,8 @@
 from unittest.mock import MagicMock
 import math
 
+pytestmark = pytest.mark.filterwarnings("ignore:RobotDrive :DeprecationWarning")
```

Comment:
Why are you saving it as a variable?

Reply:
https://docs.pytest.org/en/latest/warnings.html#pytest-mark-filterwarnings